src/Link.php

```diff
@@ -110,7 +110,7 @@ public function withHref(string|Stringable $href): self
      */
     public function withRel(string $rel): self
     {
-        if (! is_string($rel) || empty($rel)) {
+        if (! is_string($rel) || ($rel === '' || $rel === '0')) {
```

Comment:
just quick look, on `||`, parentheses on the right seems not needed

```suggestion
        if (! is_string($rel) || $rel === '' || $rel === '0') {
```

Reply:
TBH, I don't think any of the rector replacements for `empty` should be comparing to zero-string.

If the user wants a name, rel, attribute value etc of '0' that shouldn't be a problem.

Reply:
@gsteel that's needed to keep existing behaviour

Reply:
@samsonasik This is going into 3.0 though, so it doesn't matter.

Reply:
new test seems needed then

Reply:
Thanks for the feedback 👍  I’ve removed the zero-string check and added new tests